examples/xgboost_cv.py

```diff
@@ -0,0 +1,109 @@
+"""
+Optuna example that performs Cross Validation for cancer dataset
+using XGBoost.
+In this example, we perform cross-validation to accuracy of cancer detection
+using XGBoost. We optimize both the choice of booster model and their hyper
+parameters.
+We have following two ways to execute this example:
+(1) Execute this code directly.
+    $ python xgboost_cv.py
+(2) Execute through CLI.
+    $ STUDY_NAME=`optuna create-study --direction maximize --storage sqlite:///example.db`
+    $ optuna study optimize xgboost_cv.py objective --n-trials=100 --study $STUDY_NAME \
+      --storage sqlite:///example.db
+"""
+
+
+import optuna
+from optuna.samplers import TPESampler
+import os
+import sklearn.datasets
+import sklearn.metrics
+import xgboost as xgb
+
+
+# Set path to save CV results
+file_path = "./tmp_results_xgbCV_optuna/"
```

Comment:
If this is a temporary directory, how about removing it at the end of the example?
```suggestion
# Set path to save CV results
file_path = "./tmp_results_xgbCV_optuna/"
```

Reply:
Removed.

Reply:
Oh, I'm sorry, but I didn't mean to remove the code. I proposed to delete the result directory at the end of the file smilarly to the other examples. IMO, the code to save the CV results of trials will help xgboost users.

I proposed the following changes. What do you think?

```python
import os
import shutil

CV_RESULT_DIR = "./xgboost_cv_results"

if not os.path.exists(file_path):
    os.mkdir(CV_RESULT_DIR)
...

    # Save cross-validation results.
    filepath = os.path.join(CV_RESULT_DIR, '{}.csv'.format(trial.number))
    xgb_cv_results.to_csv(filepath, index=False)

...

shutil.rmtree(CV_RESULT_DIR)
```